### PR TITLE
Adding static analysis CI job

### PR DESCRIPTION
**Pull request by saadrahim**

### CODE REVIEW

File: .jenkins/common.groovy [hunk 1]
```diff
@@ -40,9 +39,6 @@ def runCompileCommand(platform, project, jobName, boolean debug=false)
             cmake -DCMAKE_BUILD_TYPE=${buildType} -DCMAKE_CXX_COMPILER=${compiler} -DCODE_OBJECT_VERSION=${cov} -DTensile_ROOT=\$(pwd)/../Tensile ../HostLibraryTests
             make -j\$(nproc)
 
-            popd
```
Comment (amdkila) [inline .jenkins/common.groovy:43]: re-add this to get rid of error when publishing results

Comment (saadrahim) [reply .jenkins/common.groovy:43]: I am just getting rid of publish results, we do not have that file.

**Comment by saadrahim** [comment]
@amdkila Can I merge now?

**Comment by amdkila** [comment]
Yes, go ahead

**Pull request merged**